python/cuopt/cuopt/tests/linear_programming/test_python_API.py

```diff
@@ -265,6 +266,52 @@ def test_constraint_matrix():
     assert rhs == exp_rhs
 
 
+def test_read_write_mps_and_relaxation():
```

Comment:
If possible lets try to cover new API that were added to set_objective_name and set_problem_name if they haven't been covered.